chirp/drivers/anytone_clone.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2023 Dan Smith <dsmith@danplanet.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from builtins import bytes
```

Comment:
The "no new uses of future" is because you're using this compatibility layer, but shouldn't.

Reply:
Oh, I didn't realize. That just carried over form the previous code and I didn't even notice it. Thanks.